components/resource_control/src/resource_group.rs

```diff
@@ -240,6 +244,14 @@ impl ResourceController {
     }
 
     pub fn update_min_virtual_time(&self) {
+        let mut guard = match self.lock.try_lock() {
+            Ok(g) => g,
+            Err(_) => {
+                // another thread is still processing, just return.
```

Comment:
I think it won't have another thread as there is a lock in `advance_min_virtual_time`

Reply:
done, removed.